src/components/TodoList/components/TodoListItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../../types/Todo';
+import { actions } from '../../../features/currentTodo';
+
+import { useAppDispatch, useAppSelector } from '../../../app/hooks';
+
+type Props = {
+  todo: Todo,
+};
+
+const TodoListItem: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+  const currentTodoId = useAppSelector(state => state.currentTodo?.id);
+  const { id: todoId, completed } = todo;
```

Comment:
You should destructure the whole todo
```suggestion
  const { id: todoId, completed, title } = todo;
```